moc_openshift.py

```diff
@@ -1,10 +1,11 @@
 """API wrapper for interacting with OpenShift authorization"""
-
+# pylint: disable=R0904
```

Comment:
If you disable things by name instead of by id it's much easier for someone reading the code to understand what you're disabling.  E.g, `# pylint: disable=too-many-public-methods`

Reply:
It may be just from previously working with lint, that I have habitually used the error number.